test/verify/check-packagekit

```diff
@@ -136,6 +138,126 @@ class TestUpdates(NoSubManCase):
         else:
             self.fail("Timed out waiting for updates spinner to go away")
 
+    @skipImage("kpatch is not available", *OSesWithoutKpatch)
+    def testKpatch(self):
+        b = self.browser
+        m = self.machine
+
+        kernel_ver_arch = m.execute("uname -r").strip()
+        sanitized_kernel_ver = "_".join(kernel_ver_arch.split(".")[:-2])
+        self.createPackage("kpatch-patch-" + sanitized_kernel_ver, "0", "0", arch=self.secondary_arch,
+                           provides="kpatch-patch = {0}".format(kernel_ver_arch))
```

Comment:
Could this also add the Requires: that the real package has, to ensure this has accurate dependencies?

Reply:
```
Requires:	kpatch >= 0.6.1-1
Requires:	kernel = %{kernel_ver}
```
The real package has these two requires. Not sure if that helps us in any way.

Reply:
Mostly as a validation that the name computation is correct, and that we are not  trying to uninstall `kpatch` while there are still some kpatches installed.

Reply:
> Mostly as a validation that the name computation is correct

How would that help? It is just `Requires: kernel = $(uname -r)` and hardcoded version of kpatch.

Reply:
The code doesn't offer to uninstall kpatch, so not important.